src/data.rb

```diff
@@ -0,0 +1,96 @@
+require 'json'
+require_relative './student'
+require_relative './rental'
+require_relative './book'
+require_relative './teacher'
+require_relative './classroom'
+
+module Data
+  def load_people
+    if File.exist? 'people.json'
+      JSON.parse(File.read('people.json')).map do |people|
+        if people['instance'] == 'teacher'
+          teacher = Teacher.new(age: people['age'], id: people['id'], name: people['name'],
+                                specialization: people['specialization'])
+          teacher
+        else
+          parent_permission = people['permission'] && true
+          student = Student.new(age: people['age'], id: people['id'], name: people['name'],
+                                parent_permission: parent_permission, classroom: people['classroom'])
+          student
+        end
+      end
+    else
```

Comment:
- [ ] when trying to run the application the user gets an error as you can see in below print screen, please fix this before requesting another review.
![Screenshot 2022-01-05 153734](https://user-images.githubusercontent.com/39240820/148235668-bc05392b-219b-4156-a176-f705ab1ddcf2.png)